special-interest-groups/sig-bigdata/README.md

```diff
@@ -5,7 +5,7 @@ BigData SIG covers the areas where TiDB and big data are combined, including but
 Anyone who focuses on big data and TiDB is welcomed to join the SIG.
 
 ## Roles and Organization Management
-See [SIG Roles and Organization Management](./roles-and-organization-management.md)
+See [SIG Roles and Organization Management](https://github.com/pingcap/community/blob/master/special-interest-groups/sig-bigdata/README.md)
```

Comment:
```suggestion
See [SIG Roles and Organization Management](https://github.com/pingcap/community/blob/master/special-interest-groups/sig-bigdata/roles-and-organization-management.md)
```

Reply:
It seems that other documents need to be adjusted as well.

Reply:
Done.